src/styles/admin/index.scss

```diff
@@ -103,7 +103,8 @@ html :where(.editor-styles-wrapper) {
 		position: sticky;
 		top: 3.5rem;
 		display: block;
-		height: 50vh;
+		height: 100vh;
+		height: 100dvh;
```

Comment:
Are there supposed to be two height values? I think the second one is just overwriting the first.

Reply:
i'm guessing this is compatibility, if the first one doesn't work then it falls back to the one that does https://caniuse.com/viewport-unit-variants

Reply:
This is correct, any time I add a 100vh, I'll throw in the 100dvh after it. The 100dvh is useful on phones where the viewport updates based on the browser dynamically resizing the height with scroll.